README.md

```diff
@@ -25,7 +25,7 @@ Documentation for all packages within this repo can be found [here](http://godoc
 ## Tutorials
 Tutorials for how to create smart contracts using packages in this repo can be found [here](./tutorials)
 
-A sample chaincode built using packages from this repo can be found in [fabric-samples](https://github.com/hyperledger/fabric-samples/tree/main/chaincode/fabcar/go)
+A sample chaincode built using packages from this repo can be found [here](https://github.com/hyperledger/fabric-contract-api-go/tree/main/integrationtest/chaincode/simple)
```

Comment:
I think it would be better to continue pointing to the fabric-samples repository since that contains many useful examples. asset-transfer-basic replaced fabcar as the initial example in the Fabric documentation:

https://github.com/hyperledger/fabric-samples/tree/main/asset-transfer-basic/chaincode-go

Reply:
Hi @bestbeforetoday , since it was simpler for me to follow the [getting started](https://github.com/hyperledger/fabric-contract-api-go/blob/main/tutorials/getting-started.md) tutorial using the simple chaincode within this repo itself, I have left it in the PR. However I have added the asset-transfer-basic chaincode as well as URLs to fabric-samples and other examples in [this repo](https://github.com/hyperledger/fabric-contract-api-go/tree/main/integrationtest/chaincode) as well.